map.go

```diff
@@ -10,6 +10,9 @@ import (
 	"github.com/cilium/ebpf/internal/unix"
 )
 
+// Specify numa node during map creation
+const NumaMode uint32 = unix.BPF_F_NUMA_NODE
```

Comment:
I'd prefer to not export the flags specifically. There is a ton of them, and maintaining them is an extra burden. Just using unix.BPF_F_NUMA_NODE directly should be fine I think.

Reply:
Agreed, I think we can add a method to explicitly set the NUMA node
```go
// Specify numa node during map creation
func (ms *MapSpec) WithNumaNode(node uint32) {
	ms.Flags |= unix.BPF_F_NUMA_NODE
	ms.NumaNode = node
}
```